pontoon/insights/static/css/insights.css

```diff
@@ -109,6 +111,42 @@
     margin-bottom: 40px;
 }
 
+#insights .suggestions-age .block {
+    position: relative;
+}
+
+#insights .suggestions-age-items {
+    width: 940px;
+    height: 212px;
+    display: inline-block;
+    overflow: hidden;
+    transition: margin 0.4s ease-in-out;
+}
+
+#insights .suggestions-age-items .suggestions-age-item {
+    float: left;
+    padding-right: 60px;
+}
+
+#insights .suggestions-age nav {
+    text-align: center;
+    position: absolute;
+    left: 0;
+    right: 0;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+#insights .suggestions-age nav ul li {
+    color: #aaaaaa;
+    cursor: pointer;
+    display: inline-block;
+}
```

Comment:
This toggle ends up _tiny_. If the intent is not to hide this toggle, it needs to be bigger. Could maybe use the same style as in the bigger activity charts?

Reply:
In the activity charts we have legends, so I worry it'll be confusing to use the same style for a different functionality. Also, chart title are long and wouldn't fit into one line.

Unless you mean we should just makes the dots bigger to match the size of the activity charts?

Reply:
I don't think it'd be enough to just make the dots bigger. I think some sort of titles are needed. While there is a difference to the activity chart toggles, I don't think it's really _all_ that different?

Reply:
<img width="481" alt="Screenshot 2021-12-01 at 14 24 25" src="https://user-images.githubusercontent.com/626716/144242258-82cd264f-216e-4d3e-aa14-4fcd68d04401.png">

Something like that? It obviously needs fine-tuning (margin etc.).

Reply:
Yes!